### PR TITLE
Remove "X" from the search field

### PR DESCRIPTION
**Pull request by huayunh**
#### Changes proposed in this Pull Request:
- Remove the browser default "X" from the search field
- Just a small thing but it bugs me a little...
- 

#### Screenshots:

Before: 

https://user-images.githubusercontent.com/8997218/149817595-4a767e15-435e-460e-9740-7273a22166b2.mov

After:

https://user-images.githubusercontent.com/8997218/149817606-a3455546-f466-417b-87ae-28462bd9286b.mov

#### To Test:
- search for something
- hover your mouse in the search field
- ensure that the x does not appear

### CODE REVIEW

**Comment by jeffvg** [comment]
I think this is a chrome thing and I don't see it in FireFox. Right now in this branch I can search without seeing the double close now but the search stops and places a "?" in the url without the entire route search "?&search=theme" weird one


https://user-images.githubusercontent.com/37150565/149823757-e52059e6-d7e9-43d0-91a4-e21a23735f90.mov

File: src/index.css [hunk 1]
```diff
@@ -34,6 +34,13 @@ q:after {
     padding: 0;
 }
 
+input[type='search']::-webkit-search-decoration,
```
Comment (emclaug2) [inline src/index.css:37]: Looks good to me.

**Comment by emclaug2** [comment]
> I think this is a chrome thing and I don't see it in FireFox. Right now in this branch I can search without seeing the double close now but the search stops and places a "?" in the url without the entire route search "?&search=theme" weird one

This is specific to firefox?  If the search is being impacted in some way could you log an issue for this?  

Approving this because the X is gone for me on Chrome.

**Review by emclaug2 — APPROVED** [review]

**Comment by huayunh** [comment]
Great catch, jeff! Removed the "form" HTML tag that is clearly not compatible with single page apps

**Pull request merged**